### ISSUE TITLE
Make fails on Cygwin

### ISSUE THREAD

**Issue by byronka** [mk270/whitakers-words#86]
...
gprbuild -j4 -Pwords meanings
Compile
   [Ada]          meanings.adb
Bind
   [gprbind]      meanings.bexch
   [Ada]          meanings.ali
Link
   [link]         meanings.adb
echo g | bin/makedict DICTLINE.GEN > /dev/null
C:/cygwin64/home/byron/dev/whitakers-words/bin/makedict.exe: error while loading shared libraries: ?: cannot open shared object file: No such file or directory
make: *** [Makefile:9: DICTFILE.GEN] Error 127

**Comment by mk270** [comment]
please try again with the most recent code; this may be a duplicate of #84

**Comment by calumapplepie** [comment]
Also note #107: maybe we should just add a note "Don't try building in parallel: our makefile will go boom"
